azuredevops/internal/acceptancetests/testutils/hcl.go

```diff
@@ -200,6 +200,24 @@ resource "azuredevops_serviceendpoint_dockerregistry" "serviceendpoint" {
 	return fmt.Sprintf("%s\n%s", projectResource, serviceEndpointResource)
 }
 
+// HclServiceEndpointAzureCRResource HCL describing an AzDO service endpoint
+func HclServiceEndpointAzureCRResource(projectName string, serviceEndpointName string /*, username string, password string*/) string {
+	serviceEndpointResource := fmt.Sprintf(`
+resource "azuredevops_serviceendpoint_azurecr" "serviceendpoint" {
+	project_id             = azuredevops_project.project.id
+	service_endpoint_name  = "%s"
+	azurecr_spn_tenantid      = "9c59cbe5-2ca1-4516-b303-8968a070edd2"
+    azurecr_subscription_id   = "3b0fee91-c36d-4d70-b1e9-fc4b9d608c3d"
+	azurecr_subscription_name = "Microsoft Azure DEMO"
+	resource_group = "testrg"
+	azurecr_name = "testacr"
+
+}`, serviceEndpointName /*, username, password*/)
```

Comment:
Remove commented out code
```suggestion
}`, serviceEndpointName)
```